src/SettingsPage/View.php

```diff
@@ -120,8 +120,6 @@ private function get_tabs() {
 	private function instantiate_tab( string $tab_class ) : ?Tabs\TabInterface {
 		if ( class_exists( $tab_class ) ) {
 			return new $tab_class( $this->settings_page );
-		} else {
-			return null;
```

Comment:
The `return null;` statement is necessary - I think.

Reply:
Looks like it. Just tested it. Just committed the fix.